streams/streams-scala/NOTICE

```diff
@@ -1,3 +0,0 @@
-Kafka Streams Scala
-Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
-Copyright (C) 2017-2018 Alexis Seigneurin.
```

Comment:
I'm pretty sure there's not supposed to be extra NOTICE files in sub-directories of the project.

Reply:
It would definitely not be common/standard